raiden/raiden_service.py

```diff
@@ -982,21 +982,29 @@ def _initialize_channel_fees(self) -> None:
 
             for channel in channels:
                 # get the flat fee for this network if set, otherwise the default
-                flat_fee = fee_config.get_flat_fee(channel.token_network_address)
+                flat_fee = fee_config.get_flat_fee(channel.token_address)
+                proportional_fee = fee_config.get_proportional_fee(channel.token_address)
+                proportional_imbalance_fee = fee_config.get_proportional_imbalance_fee(
+                    channel.token_address
+                )
                 log.info(
                     "Updating channel fees",
                     channel=channel.canonical_identifier,
                     flat_fee=flat_fee,
-                    proportional_fee=fee_config.proportional_fee,
-                    proportional_imbalance_fee=fee_config.proportional_imbalance_fee,
+                    proportional_fee=proportional_fee,
+                    proportional_imbalance_fee=proportional_imbalance_fee,
+                )
+                proportional_imbalance_fee = fee_config.get_proportional_imbalance_fee(
```

Comment:
Why request not use the above? Seems like this can be removed.

Reply:
nicely spotted. Probably mistakes due to the rebase. These lines had conflicts every time.